spacy/cli/train.py

```diff
@@ -50,14 +56,4 @@ def train_cli(
         nlp = init_nlp(config, use_gpu=use_gpu)
     msg.good("Initialized pipeline")
     msg.divider("Training pipeline")
-    train(nlp, output_path, use_gpu=use_gpu, silent=False)
-
-
-def verify_cli_args(config_path: Path, output_path: Optional[Path] = None) -> None:
```

Comment:
Are we going to revert all this stuff again now? If so, we should probably do this consistently across all the other functions that we changed based on your initial requests

Reply:
As we've removed almost all of the CLI args from that function, it felt a bit better to just inline it again. I wouldn't go through and make the other changes, I don't think it's important that that's consistent. I can revert it if you'd prefer they were all the same though.